.github/workflows/sanity-test.yml

```diff
@@ -190,6 +190,29 @@ jobs:
       - name: Build aws-c-common
         run: python3 builder.pyz build --project aws-c-common --compiler=${{ matrix.compiler }} run_tests=false
 
+  release_notes:
+    strategy:
+      fail-fast: false
+      matrix:
+        project:
+          - aws-crt-cpp
+          - aws-iot-device-sdk-cpp-v2
+          - aws-crt-java
+          - aws-iot-device-sdk-java-v2
+          - aws-crt-nodejs
+          - aws-iot-device-sdk-js-v2
```

Comment:
Trivial: This script is only useful for projects that use submodules.
Remove `aws-iot-device-sdk-java-v2` and `aws-iot-device-sdk-js-v2`
Add `aws-crt-python`

Reply:
I've added python, but this is just sanity tests. It should *work* anywhere. Whether the output is useful 🤷‍♂️ , but this is to make sure there isn't a regression when there aren't submodules, or something else.